client/src/app.jsx

```diff
@@ -1,57 +1,82 @@
-import React from 'react';
-import { Container, Row } from 'react-bootstrap';
+import React, { useState, useEffect } from 'react';
+import {
+  Container, Row, Accordion, Col,
+} from 'react-bootstrap';
```

Comment:
the bootstrap docs recommend you import each component with the specific reat-bootstrap/X file path to reduce the overall code you're sending a client https://react-bootstrap.github.io/getting-started/introduction/

Reply:
Nice! Updated all my files to adhere to the docs :D